### PR TITLE
Update README and prepare for release

### PR DESCRIPTION
**Pull request by mkostick**
This is a companion pull to #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,250 @@ Or install it yourself as:
...
+      # matching handler in the reverse order of their definitions.
+      rescue_from StandardError do |ex|
+        render json: { error: ex.message }, status: 500
+      end
+
+      # For the YetiException exceptions with richer data, use the specified
+      # details and status.
+      rescue_from YetiException::Error do |ex|
+        render json: { error: ex.details }, status: ex.status
+      end
```
Comment (dmills-yesware) [inline README.md:195]: [suggestion] Let's log the exceptions in these examples. I realize you showed logging examples above, but devs are going to use these snippets directly, and few things drive me crazier than unlogged errors.

File: README.md [hunk 2]
```diff
@@ -18,8 +18,250 @@ Or install it yourself as:
...
+      # Define a catch-all handler for other exceptions.  Rails searches for a
+      # matching handler in the reverse order of their definitions.
+      rescue_from StandardError do |ex|
+        render json: { error: ex.message }, status: 500
+      end
+
+      # For the YetiException exceptions with richer data, use the specified
+      # details and status.
+      rescue_from YetiException::Error do |ex|
+        render json: { error: ex.details }, status: ex.status
```
Comment (dmills-yesware) [inline README.md:194]: Just to be clear: The stacktrace isn't part of `details`, right? Any chance of there being security-sensitive data in there by default?

**Review by dmills-yesware — COMMENTED** [review]
Just the one question and the one suggestion. Looks great 👍

**Review by mattduggan — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mkostick) [reply README.md:194]: The stack trace is not part of the details.  The details only ever contain:
- the hash passed in by the user
- the return value of the `#msg` method implemented by subclasses
- the original exception's message, in the case of a wrapped exception

**Review by faustus7 — APPROVED** [review]

**Comment by mkostick** [comment]
Updated to add yeti_logger and to remove wrap_exception (see #2)

**Review by dmills-yesware — APPROVED** [review]
Thanks for the updates. LGTM 👍

**Pull request merged**